composer.json

```diff
@@ -22,7 +22,7 @@
         }
     },
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.4"
```

Comment:
I understand, thank you for providing the patch.

There is no new BC-breaking version planned since this is just a minor thing and a PHP implementation is in the works for 7.1, so I'll be closing this. Users can still implement this in sub-classes, but I don't see a big value since it doesn't support deserialization.